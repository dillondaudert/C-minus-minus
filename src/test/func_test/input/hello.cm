int main() {
	write('Hello world!');
	exit;
}
