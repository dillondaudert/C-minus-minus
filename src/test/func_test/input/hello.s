	.section	.rodata
.int_wformat: .string "%d\n"
.str_wformat: .string "%s\n"
.int_rformat: .string "%d"

.string_const0: .string "Hello world!"

	.globl main
	.type main @function

main:	nop
	.size	main, .-main
	leave
	ret
