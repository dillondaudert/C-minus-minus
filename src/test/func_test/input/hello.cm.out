st_init created the symbol table hash map
Adding new symbol: main, main, offset: 0, type: 2, size: 4, arr: 0
st_add added main at position 0
Added process main to stable

Adding new symbol: .string_const0, .string_const0, offset: 0, type: 0, size: 13, arr: 0
st_add, added .string_const0 to collision LL at 0



Freed symbol at position 0
Freed LL symbol at position 0
Freed link in collision LL at 0
Freed symbol table
